spec/mailers/user_mailer_spec.rb

```diff
@@ -432,4 +468,77 @@
       end
     end
   end
+
+  describe "invited to collection" do
+    let!(:gift_exchange) { create(:gift_exchange) }
+    let!(:collection) { create(:collection, challenge: gift_exchange, challenge_type: "GiftExchange") }
+    let!(:otheruser) { create(:user) }
```

Comment:
Is this meant to be the creator of the work, since they're the person who would be receiving this email? Also, I think just plain `let` _might_ work here -- wouldn't everything be created when we send the email on line 478?

Reply:
Yes, they would normally be the creator of the work, I'll add that link in. Let previously failed but might work now we've removed the gift exchange complications.